frame-component/src/index.ts

```diff
@@ -1,3 +1,12 @@
-export function frameComponent() {
-  // TODO
+import { ChildComponent, ChildProps } from "./child-component";
+import { ParentComponent, ParentProps } from "./parent-component";
+import { uniqueID } from "./utils";
+
+export function parent(options: ParentProps): ParentComponent {
+  return new ParentComponent(options);
 }
+
+export function child(options: ChildProps): ChildComponent {
```

Comment:
Can we name these `createParent` and `createChild`

```suggestion
export function createParent(options: ParentProps): ParentComponent {
  return new ParentComponent(options);
}

export function createChild(options: ChildProps): ChildComponent {
```